### PR TITLE
Update custom-roles.md

### PR DESCRIPTION
**Pull request by msbemba**
The first column should represent the work load and not the permissions

Fixes #https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10370

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender/custom-roles.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ The following table outlines the roles and permissions required to access each u
 > [!NOTE]
 > Incident management requires management permissions for all products that are part of the incident.
  
-| **One of the following roles are required for Microsoft 365 Defender**  | **One of the following roles are required for Defender for Endpoint**  | **One of the following roles are required for Defender for Office 365** | **One of the following roles are required for Defender for Cloud Apps** | 
+| **M365 defender workload**  | **One of the following roles are required for Defender for Endpoint**  | **One of the following roles are required for Defender for Office 365** | **One of the following roles are required for Defender for Cloud Apps** |
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/custom-roles.md]:
```suggestion
| **Microsoft 365 Defender workload**  | **One of the following roles is required for Defender for Endpoint**  | **One of the following roles is required for Defender for Office 365** | **One of the following roles is required for Defender for Cloud Apps** | 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Pull request merged**